fixtures/go.exoskeleton

```diff
@@ -3,7 +3,6 @@
 if [[ "$1" == "--describe-commands" ]]; then
   cat << EOF
 {
-  "name": "go",
```

Comment:
Removing this caused a test to fail that removing line 50 from `executable_module.go` fixed